es6-modules/src/components/todo-list.js

```diff
@@ -0,0 +1,54 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib/create-element'
```

Comment:
Here it would be better to import from the `lib/index.js` file:

```JAVASCRIPT
import { createElement } from '../lib';
```

Reply:
Updated